cicecore/cicedynB/general/ice_init.F90

```diff
@@ -1023,6 +1025,16 @@ subroutine input_data
          endif
       endif
 
+      if (grid_ice == 'C' .or. grid_ice == 'CD') then
+         if (visc_coeff_method /= 'avg_zeta' .and. visc_coeff_method /= 'avg_strength') then
```

Comment:
Maybe we want to use `trim` here, for safety ? that seems to be the custom in ice_init...